notebooks/dc2/truth_variable_info.ipynb

```diff
@@ -0,0 +1,520 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sn_summary_cat = GCRCatalogs.load_catalog(\"desc_dc2_run2.2i_truth_sn_summary\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted(sn_summary_cat.list_all_quantities())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now actually load the data into memory. For this tutorial, we will only need the ID (`id`), coordinates (`ra`, `dec`), redshift. \n",
+    "\n",
+    "Recall that `get_quantities` returns a Python dictionary. We can use `pd.DataFrame` to turn it into a Pandas DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d_summary = pd.DataFrame(sn_summary_cat.get_quantities([\"id\", \"ra\", \"dec\", \"redshift\"]))\n",
+    "d_summary.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have the summary table, we can use it to select, say supernovae near a certain coordinate or in a certain redshift range. In this tutorial, we will find all supernovae that are within 2 arcmin of (RA, Dec) = (60.5, -36.6) and within z < 0.8. We will store these supernovae in `objects_to_plot` which we will use later to extract and plot the their light curves."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "selected = SkyCoord(d_summary[\"ra\"].values, d_summary[\"dec\"].values, unit=\"deg\").separation(SkyCoord(60.5, -36.6, unit=\"deg\")).arcmin < 2\n",
+    "selected &= d_summary[\"redshift\"] < 0.8\n",
+    "\n",
+    "objects_to_plot = d_summary.iloc[np.flatnonzero(selected)]\n",
+    "objects_to_plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading supernova light curve (variability) table\n",
+    "\n",
+    "Now that we have a list of supernovae that we want to inspect further, we can load the \"variability table\", which contains the light curves. \n",
+    "\n",
+    "We will load the \"variability table\" into `sn_light_curve_cat` and check what columns are available. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sn_light_curve_cat = GCRCatalogs.load_catalog(\"desc_dc2_run2.2i_truth_sn_variability\")\n",
+    "sn_light_curve_cat.list_all_quantities()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unlike the summary table, in the variability table, each row corresponds to one \"measurement\" the flux of a given object, at a given time, with a given filter. Hence, a single supernovae may correspond to many rows. \n",
```

Comment:
1. Should not be a comma after variability table.  2. There is a missing word ("of") or missing punctuation (colon) after "measurement".   3. "corresponds" is maybe not the best word here.   I suggest

Unlike the summary table, in the variability table each row corresponds to one "measurement": the flux of a given object, at a given time, with a given filter. Hence, a single supernova is usually represented by many rows.